endpoint/Makefile

```diff
@@ -0,0 +1,82 @@
+ifeq ($(ARM), y)
+CROSS = arm-linux-gnueabi-
```

Comment:
This is legato-specific. You should leave some informative comments starting with `FIXME` and/or `TODO`.